StudyRingMeteor/client/rings/RingDash.js

```diff
@@ -27,17 +27,23 @@ Template.RingDash.helpers({
         return Meteor.users.find({rings: id});
     },
     comments: ()=> {
-      var active = Session.get('activeRing');
+      var ringId = FlowRouter.getParam('id');
+      console.log(ringId);
+      Session.set('dashActive',ringId);
+      var active = Session.get('dashActive');
       return UserDiscussion.find({ringId: active});
```

Comment:
You can pass in the ringId from line 30 directly into the query instead of setting and getting a session variable.